types & grammar/ch2.md

```diff
@@ -115,7 +115,7 @@ var a = "foo";
 var b = ["f","o","o"];
 ```
 
-Строки имеют поверхностные сходства по отношению к массивам и массивоподобным, такие как -- например, оба из них имеют свойство `length`, метод `indexOf(..)` (`array` только в ES5), и метод `concat(..)`:
+Строки имеют поверхностные сходства по отношению к массивам -- и массивоподобным, как выше -- например, оба из них имеют свойство `length`, метод `indexOf(..)` (`array` только в ES5), и метод `concat(..)`:
```

Comment:
[Original](https://github.com/getify/You-Dont-Know-JS/blame/master/types%20%26%20grammar/ch2.md#L118):

> Strings do have a shallow resemblance to `array`s -- `array`-likes, as above -- for instance, both of them ...